packages/library/cozy-tls/Makefile

```
@@ -0,0 +1,8 @@
+include ../../../hack/common-envs.mk
+include ../../../hack/package.mk
+
+generate:
+	cozyvalues-gen -v values.yaml -s values.schema.json -r README.md
+
+test:
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)